src/main/java/com/amihaiemil/zold/ZoldWts.java

```diff
@@ -32,5 +32,9 @@
  * @since 0.0.1
  */
 public interface ZoldWts {
+    /**
+    * Pull the wallet from the network.
+    */
+    void pull();
```

Comment:
@ammaratef45 return ``Wallet`` here please :)

Reply:
done in the next commit 👍